src/components/CurrentIngredient.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class CurrentIngredient extends React.Component {
+  constructor(props) {
+    super(props);
+    this.deleteIngredient = this.deleteIngredient.bind(this);
+  }
+
+  deleteIngredient() {
+    this.props.onDeleteIngredient(this.props.index);
+  }
+
+  render() {
+    return (
+     <li key={this.props.index}>
+     <button id='delete-ingredient' onClick={this.deleteIngredient}>x </button>
```

Comment:
Is there supposed to be a space after the x? Just thinking it might do funny things when trying to style it.

Reply:
There is a space because before it was a href link and without that space, it would have been directly connected to the ingredient. Now it is a button so it is not needed.
Thanks for noticing it.